usecases/monitoring/prometheus.go

```diff
@@ -57,19 +57,16 @@ type PrometheusMetrics struct {
 }
 
 var (
-	msBuckets = []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 25, 50, 100, 250, 500, 1000}
-	metrics   *PrometheusMetrics
+	msBuckets                    = []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 25, 50, 100, 250, 500, 1000}
+	metrics   *PrometheusMetrics = NewPrometheusMetrics()
```

Comment:
Maybe we should put that metrics initilisation inside `init` function:
```
var metrics *PrometheusMetrics

func init() {
  metrics = NewPrometheusMetrics()
}
```
We would then use golang's built-in function to explicitly initialise metrics variable.

Reply:
I implemented using init() but I changed because this version.
Basically they are both built-in into the language but this version is short.

Reply:
Yes, I know it's shorter, but was wondering if making it inside `init()` function would be more verbose to someone reading the code that we do here in this function explicit initialisation of metrics variable

Reply:
I think the current version is more readable than what you are suggesting. It is short and is in the same initialization block as other variables.
Having said that I it changed back to use init() to make you happy.